app/graphql/types/volume.rb

```diff
@@ -0,0 +1,37 @@
+class Types::Volume < Types::BaseObject
+  description 'Multiple Chapters of a Manga'
+
+  field :id, ID, null: false
+
+  field :titles, Types::TitlesList,
+    null: true,
+    description: 'the Titles for this unit in various locales'
+
+  def titles
+    {
+      localized: object.titles,
+      canonical: object.canonical_title
+    }
+  end
+
+  field :number, Integer,
+    null: false,
+    description: 'The sequence number of this unit'
+
+  field :thumbnail, Types::Image,
+    null: true,
+    description: 'A thumbnail image for the unit'
+
+  field :chapters_count, Integer,
+    null: false,
+    description: 'Total chaters per Volume'
+
+  field :isbn, String,
+    null: false,
+    description: 'Unique Identification for this Volume'
```

Comment:
1. This is an array of ISBN numbers (so `field :isbn, [String]`)
2. The description doesn't help much (they're ISBN numbers so they actually identify a specific release of the volume, but the description doesn't explain that)

Reply:
Yeah ISBN numbers are a bit weird.  We don't use them internally to identify things, but they're useful for things like finding physical copies for sale